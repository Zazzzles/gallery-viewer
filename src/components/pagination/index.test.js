import React from 'react';
import renderer from 'react-test-renderer';

import Pagination from './index';

it('renders correctly', () => {
  const tree = renderer
    .create(<Pagination pageCount={1} activePage={1} />)
    .toJSON();
  expect(tree).toMatchSnapshot();
});
