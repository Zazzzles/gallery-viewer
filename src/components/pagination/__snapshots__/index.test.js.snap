// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="Container"
>
  <div
    className="Wrapper"
    style={
      Object {
        "width": "40px",
      }
    }
  >
    <div
      className="MenuDot"
    />
  </div>
</div>
`;
